examples/fabric/nested_ingestion_pipeline/README.md

```diff
@@ -0,0 +1,96 @@
+# Nested pipelines
+
+The following example, exemplifies the utilization of nested pipelines including control activities like ForEach and If.
+
+![Nested Pipelines](pl_main_pipeline.png)
+
+The main pipeline is composed of 2 top activities:
+
+- a lookup activity named "Read Configuration File
```

Comment:
```suggestion
- a Lookup activity named "Read Configuration File
```

Reply:
I've also added the closing quote that was missing.